taxi/views.py

```diff
@@ -87,3 +100,33 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+# ------Car-Driver------------------------------------------------------------
+
+
+@login_required
+def assign_user_to_car(request, pk):
+    car = get_object_or_404(Car, pk=pk)
+
+    if request.user not in car.drivers.all():
+        car.drivers.add(request.user)
+    else:
+        car.drivers.remove(request.user)
+
+    return redirect("taxi:car-detail", pk=pk)
```

Comment:
The `assign_user_to_car` function toggles the user's assignment to a car. Consider renaming the function to something like `toggle_user_assignment_to_car` to better reflect its functionality. Additionally, ensure that the logic for adding or removing a user is clearly documented, as it might be confusing to users or developers unfamiliar with the code.